turbinia/workers/analysis/yara.py

```diff
@@ -69,39 +73,56 @@ def run(self, evidence, result):
     result.close(self, success=True, status=summary)
     return result
 
-  def runLoki(self, result, evidence):
-    log_file = os.path.join(self.output_dir, 'loki.log')
-    stdout_file = os.path.join(self.output_dir, 'loki_stdout.log')
-    stderr_file = os.path.join(self.output_dir, 'loki_stderr.log')
+  def runFraken(self, result, evidence):
+    stdout_file = os.path.join(self.output_dir, 'fraken_stdout.log')
 
     cmd = [
-        'sudo', 'python', '/opt/loki/loki.py', '-s', '10000', '--csv',
-        '--intense', '--noprocscan', '--dontwait', '--noindicator',
-        '--nolevcheck', '--nolisten', '-l', log_file, '-p', evidence.local_path
+        'sudo', '/opt/fraken/fraken', '-rules', '/opt/signature-base/',
+        '-folder', evidence.local_path
     ]
 
-    (ret, result) = self.execute(
-        cmd, result, log_files=[log_file], stdout_file=stdout_file,
-        stderr_file=stderr_file, cwd='/opt/loki/')
+    yr = self.task_config.get('yara_rules', '')
+    if yr != '':
```

Comment:
Is it possible this could be `None`?  Maybe do `if not yr`?

Reply:
Done